boards/st/stm32f769i_disco/stm32f769i_disco.dts

```diff
@@ -135,19 +130,6 @@ arduino_serial: &usart6 {};
 	clock-frequency = <I2C_BITRATE_FAST>;
 };
 
-&i2c4 {
```

Comment:
If I understand right, this display should be defined as a shield now to get the same functionality on the board? If so, we should put a note in the migration guide for 4.0 highlighting that a shield now needs to be set to use a display with this board for downstream users who are developing with this board and expect a display device to be defined.

Reply:
there is no on-board display/touch and display support is added as new for this board using shield (updated in the board doc), I doubt this needs to me mentioned in migration guide. please correct me if my understanding is wrong.

Reply:
@danieldegrasse This i2c node was added in https://github.com/zephyrproject-rtos/zephyr/pull/41519 to support an external touch controller.
In any case, the touch controller child node shouldn't be in the board's DTS, instead it should be in the overlay of each supported touchscreen shield.